src/compile.rs

```diff
@@ -65,7 +65,7 @@ fn handle_module(
         let mut mem = vec![];
 
         // Compile and encode the result to machine code.
-        let (code_size, rodata_size) = context
+        let offsets = context
```

Comment:
nit: maybe rename `offsets` to `code_info`?

Reply:
Done.